src/lib.rs

```diff
@@ -737,6 +797,54 @@ pub fn write_stop_points_to_csv<P: AsRef<Path>>(
     }
 }
 
+pub fn write_station_accesses_to_csv<P: AsRef<Path>>(
+    station_accesses: &[StationAccess],
+    output_dir: P,
+    all_tags: bool,
+) {
+    let output_dir = output_dir.as_ref();
+    let csv_file = output_dir.join("osm-transit-extractor_station_accesses.csv");
+
+    let mut wtr = csv::Writer::from_path(csv_file).unwrap();
+    let default_header = ["station_access_id", "lat", "lon", "name", "code"];
+    let osm_tag_list: BTreeSet<String> = station_accesses
+        .iter()
+        .flat_map(|s| s.all_osm_tags.keys().map(|s| s.to_string()))
+        .collect();
```

Comment:
You can change that to the following (no need to allocate for `String` since this is done another time later).

```suggestion
    let osm_tag_list: BTreeSet<_> = station_accesses
        .iter()
        .flat_map(|s| s.all_osm_tags.keys())
        .collect();
```

Reply:
I've pushed ee8540e to fix that one. The rest of the PR is good for me.